File: kattis/telja/telja.py
import sys

class telja:

    def __init__(self, debug: bool = False):
        self.debug = debug

    def dprint(self, *args, **kwargs):
        if self.debug:
            print(*args, **kwargs)

    def generateOutput(self):
        # Take input from user
        self.dprint("-------------------------------------")
        self.dprint("Input the number.")
        inputNumber = sys.stdin.readline()
        if not inputNumber:
            return

        number = int(inputNumber.strip())
        self.dprint(f"The number is: {number}")

        # Count upto the input number sequentially in different line
        self.dprint("-------------------------------------")
        # Approach 1
        # for i in range(1, number+1):
            # print(i)

        # Approach 2
        for i in range(number):
            print(i + 1)
        self.dprint("-------------------------------------")

if __name__ == "__main__":
    telja(False).generateOutput()

File: kattis/telja/telja.java
package kattis.telja;

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class telja {

    private final boolean debug;
    private final BufferedReader bufferedReader;
    private StringTokenizer stringTokenizer;

    public telja(boolean debug) {
        this.debug = debug;
        this.bufferedReader = new BufferedReader(new InputStreamReader(System.in));
    }

    private void dprint(Object... args) {
        if (debug) {
            StringBuilder sb = new StringBuilder();
            for (Object arg : args) {
                sb.append(arg).append(" ");
            }
            System.out.println(sb.toString().trim());
        }
    }

    private String nextToken() throws IOException {
        while (stringTokenizer == null || !stringTokenizer.hasMoreTokens()) {
            String line = bufferedReader.readLine();
            if (line == null)
                return null;
            stringTokenizer = new StringTokenizer(line);
        }
        return stringTokenizer.nextToken();
    }

    public void generateOutput() {
        try {
            // Take input from user
            dprint("-------------------------------------");
            dprint("Input the number.");
            String inputNumber = nextToken();
            if (inputNumber == null)
                return;

            int number = Integer.parseInt(inputNumber);
            dprint("The number is:", number);

            // Count upto the input number sequentially in different line
            dprint("-------------------------------------");
            for (int i = 0; i < number; i++){
                System.out.println(i+1);
            }
            dprint("-------------------------------------");
        } catch (Exception exception) {
            if (debug)
                exception.printStackTrace();
        }
    }

    public static void main(String[] args) {
        new telja(false).generateOutput();
    }
}